malaria24/ona/tests/test_models.py

```diff
@@ -496,6 +496,145 @@ def test_send_digest_email(self):
+        Facility.objects.create(facility_code='342315',
+                                facility_name='Facility 1',
+                                province='Limpopo',
+                                district=u'Example1')
+        Facility.objects.create(facility_code='343333',
+                                facility_name='Facility 2',
+                                province='Limpopo',
+                                district=u'Example2')
+        Facility.objects.create(facility_code='345555',
+                                facility_name='Facility 1',
+                                province='Limpopo',
+                                district=u'Example6')
+        Facility.objects.create(facility_code='343333',
+                                facility_name='Facility 2',
+                                province='Limpopo',
+                                district=u'Example2')
+        Facility.objects.create(facility_code='333444',
+                                facility_name='Facility 3',
+                                province='Limpopo',
+                                district=u'Example3')
+        Facility.objects.create(facility_code='333441',
+                                facility_name='Facility 4',
+                                province='Limpopo',
+                                district=u'Example3')
+        Facility.objects.create(facility_code='333333',
+                                facility_name='Facility 3',
+                                province='The Eastern Cape',
+                                district=u'Example7')
+        Facility.objects.create(facility_code='111222',
+                                facility_name='Facility 2',
+                                province='Gauteng',
+                                district=u'Example5')
+        Facility.objects.create(facility_code='111111',
+                                facility_name='Facility 1',
+                                province='Gauteng',
+                                district=u'Example5')
+        Facility.objects.create(facility_code='333333',
+                                facility_name='Facility 3',
+                                province='The Eastern Cape',
+                                district=u'Example7')
+        self.mk_actor(role=MANAGER_NATIONAL,
+                      email_address='manager@example.org')
+        ehp1 = self.mk_ehp(name='EHP1', email_address='ehp1@example.org')
+        ehp2 = self.mk_ehp(name='EHP2', email_address='ehp2@example.org')
+        self.mk_mis(name='MIS', email_address='mis@example.org')
+
+        for i in range(10):
+            case = self.mk_case(gender='female', facility_code='342315')
+            case.date_of_birth = datetime.today().strftime("%y%m%d")
+            case.ehps.add(ehp1)
+            case.ehps.add(ehp2)
+            case.save()
+            case.digest = None
+
+        for i in range(10):
+            case = self.mk_case(gender='male', facility_code='222222')
```

Comment:
Cases without a facility again. Do these get displayed in the digest? I _think_ you don't need them.